lib/scripts/graphics3d/volumeFilter3d.js

```diff
@@ -22,9 +22,12 @@ under the License.
  * @module lib/scripts/graphics3d/volumeFilter3d
  */
 
+
 import * as THREE from 'three';
 import MaterialBlur from '../gfx/matblur';
 import MaterialAO from '../gfx/matAO';
+import { tools3d } from '../../../app/scripts/menu';
```

Comment:
`Menu` is an add-on module here, so it would be better to remove this import. Since you use `tools3d` more as an eraser mode then as a 3d tool type, you may declare here `eraserMode` const.